test/parallel/test-runner-run.mjs

```diff
@@ -8,6 +8,11 @@ import assert from 'node:assert';
 const testFixtures = fixtures.path('test-runner');
 
 describe('require(\'node:test\').run', { concurrency: true }, () => {
+  it('should fail when passing array as options', () => {
+    assert.throws(() => run([]), {
```

Comment:
Can you add more (number, bigint, symbol, string, boolean) please?

Reply:
done